experimental/uefi/README.md

```diff
@@ -1,38 +1,37 @@
 # Experimental UEFI app and untrusted loader
 
-This directory contains a bare-bones UEFI app and an "untrusted loader" built
-around `qemu` that communicates with said app. The goal of the experiment is to
-try out different communication mechanisms etc to see how Oak could run in a
-trusted environment as an UEFI app with minimal dependencies, with code that
-doesn't need to be trusted left outside the trusted environment.
+This directory contains the following crates:
 
-To run:
+- `runtime`: common business logic that should run in a VM
+- `app`: UEFI application that wraps `runtime` for execution under UEFI
```

Comment:
Note that these are not actually the crate name; that would be this:

https://github.com/project-oak/oak/blob/3268b34f431f9ecf57d16275d980ae30ccbf942c/experimental/uefi/app/Cargo.toml#L2

BTW I think there are inconsistencies with the naming already, may be worth fixing them.

In fact we should actually avoid nesting crates into other folders, since crates actually live in a global namespace, so the directory nesting is actually more confusing than helpful. See https://github.com/project-oak/oak/issues/2533

Reply:
Fair enough, I've stopped calling them crates. :)

I think right now it makes sense to keep them under experimental, at least until we know which platforms we are targeting.